pom.xml

```diff
@@ -88,7 +88,9 @@
     <slf4j.version>1.7.10</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
     <libthrift.version>0.9.2</libthrift.version>
+    <governator.version>1.14.2</governator.version>
```

Comment:
Seems like governator is not used (all import are commented), I think it better to remove it instead of having bunch of commented code.